stdlib/REPL/src/docview.jl

```diff
@@ -229,7 +229,8 @@ repl_search(s) = repl_search(stdout, s)
 
 function repl_corrections(io::IO, s)
     print(io, "Couldn't find ")
-    printstyled(io, s, '\n', color=:cyan)
+    pre = context_module == Main ? [] : [context_module, '.']
```

Comment:
Here I tried to mimic the behaviour introduced in #23806. It's not exactly same, but kind of consistent IMO.